index.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <header class="header">
+      <nav class="header_nav">
+        <div class="logo black">
+          <a class="black" href="#">Logo</a>
+        </div>
+        <div class="header_menu">
+          <img class="menu_icon" src="assets/ic_menu.png" alt="menu" />
+        </div>
+      </nav>
+    </header>
+    <section class="headline">
+      <h1 class="headline_title">
+        <span class="sub-headline_name">Hey there, I'm Arslan</span>
+        <span class="sub-headline">I'm a software developer</span>
+      </h1>
+      <p class="headline_content">
+        I can help you build a product , feature or website Look through some of
+        my work and experience! If you like what you see and have a project you
+        need coded, don,t hestiate to contact me.
+      </p>
+      <div class="backimg"></div>
+      <div class="addline_social">
+        <ul class="social_links">
+          <li class="social_link">
+            <a href="#">
+              <img
+                class="social_icon"
+                src="assets/ic_twitter.svg"
+                alt="Twitter"
+              />
+            </a>
+          </li>
+          <li class="social_links"></li>
+          <li class="social_link">
+            <a href="#">
+              <img
+                class="social_icon"
+                src="assets/ic_linkedIn.svg"
+                alt="LinkedIn"
+              />
+            </a>
+          </li>
+          <li class="social_link">
+            <a href="#">
+              <img
+                class="social_icon"
+                src="assets/ic_github.svg"
+                alt="Github"
+              />
+            </a>
+          </li>
+          <li class="social_link">
+            <a href="#">
+              <img
+                class="social_icon"
+                src="assets/ic_medium.svg"
+                alt="Medium"
+              />
+            </a>
+          </li>
+          <li class="social_link">
+            <a href="#">
+              <img
+                class="social_icon"
+                src="assets/ic_angellist.svg"
+                alt="AngelList"
+              />
+            </a>
+          </li>
+        </ul>
```

Comment:
- [ ] **Required:** Your social icons are not properly centered, and it seems to be due to an empty `li` element which causes your icons to be off. You could look for the empty `li` element delete it to fix the alignment

Figma design
![image](https://user-images.githubusercontent.com/50066876/236192082-d4f185fe-c839-4acc-85bf-efa7c1f19986.png)


Your design
![image](https://user-images.githubusercontent.com/50066876/236192001-b0c2078a-030d-4bd8-a9ab-4652f083c681.png)